src/Workspaces/Core/Portable/CodeRefactorings/FixAllOccurences/FixAllContext.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixesAndRefactorings;
+using Microsoft.CodeAnalysis.Shared.Utilities;
+using Microsoft.CodeAnalysis.Text;
+using FixAllScope = Microsoft.CodeAnalysis.CodeFixes.FixAllScope;
+
+namespace Microsoft.CodeAnalysis.CodeRefactorings
+{
+    /// <summary>
+    /// Context for "Fix all occurrences" for code refactorings provided by each <see cref="CodeRefactoringProvider"/>.
+    /// </summary>
+    /// <remarks>
+    /// TODO: Make public, tracked with https://github.com/dotnet/roslyn/issues/60703
+    /// </remarks>
+    internal sealed class FixAllContext : IFixAllContext
```

Comment:
in the future (when we're getting ready to make public), it would be nice to have an abstract base type (with a `private protected` constructor) that we can move common code up into.